src/Nokia_LCD.cpp

```diff
@@ -14,12 +14,17 @@
 namespace {
 const uint8_t kDisplay_max_width = 84;
 const uint8_t kDisplay_max_height = 48;
+
+const Nokia_LCD_Fonts::DefaultFont defaultFont;
+
 // Each row is made of 8-bit columns
-const unsigned int kTotal_rows =
-    kDisplay_max_height / Nokia_LCD_Fonts::kRows_per_character;
+unsigned int kTotal_rows =
+    kDisplay_max_height / defaultFont.rowSize();
```

Comment:
We should not have global variables that are _not_ constants. Before we turn it into a (non-constant) member variable, does it make sense for the total rows to change or be different than the default? Is there a use case for this?

Reply:
You're right. We don't have fonts with more than one row height. So, no problem in keeping this fixed to `Nokia_LCD_Fonts::kRows_per_character`